legacy/populate_db.py

```diff
@@ -89,70 +73,6 @@ def save_measurements_to_database(conn, yml_dir, submission_form, data_template)
     num_comu         = len(info_comu['Community_ID'])
     num_rep_metadata = len(replicate_metadata['Biological_Replicate_id'])
 
-    if 'Study_UniqueID' not in info_study:
-        errors.append("Missing study unique ID in the study template")
-    elif info_study['Study_UniqueID'][0].strip() != submission.studyUniqueID:
-        errors.append(
-            "The unique ID for the Study in the form doesn't match the one in the uploaded Excel file. "
-            "Check if you've uploaded the wrong file."
-        )
-
-    if 'Project_UniqueID' not in info_study:
-        errors.append("Missing study unique ID in the study template")
-    elif info_study['Project_UniqueID'][0].strip() != submission.projectUniqueID:
-        errors.append(
-            "The unique ID for the Project in the form doesn't match the one in the uploaded Excel file. "
-            "Check if you've uploaded the wrong file."
-        )
```

Comment:
I've removed the check for project/study UUID with the spreadsheet. The study spreadsheet will be removed anyway, so it won't be necessary. And, at the moment, it's making it harder to test different studies with the same spreadsheet.

Reply:
Yes, the `study_id` will be `SMGDB00...x` and `study_uuid` is the private id.

In the long term, I'd like to consistently name these `publicId` and `uuid`, respectively, in studies, projects, and users.